runtime/help/options.md

```diff
@@ -415,7 +415,7 @@ Here are the available options:
 * `statusformatl`: format string definition for the left-justified part of the
    statusline. Special directives should be placed inside `$()`. Special
    directives include: `filename`, `modified`, `line`, `col`, `lines`,
-   `percentage`, `opt`, `bind`.
+   `percentage`, `opt`, `overwrite`, `bind`.
```

Comment:
Maybe even add it to the default values? Otherwise we've to refer again to the doc that there is a customizable flag.

Reply:
Yep, as I also suggested in https://github.com/zyedidia/micro/pull/3620#discussion_r1922884053.

Reply:
Overseen. :wink: